homework/burgers/views/order.handlebars

```diff
@@ -1,34 +1,37 @@
-<div class="page-header">
-  <h1>Jessica's Burgers<small>{{ptitle}}</small></h1>
-</div>
+<h1><a href="/">Jessica's Burgers</h1></a>
 <form class="form-horizontal">
   <fieldset>
 
     <!-- Form Name -->
     <legend>Build your burger!</legend>
+    <legend>Order total: $<span class='orderprice'>0.00</span></legend>
 
     <!-- Multiple Checkboxes -->
-    <div class="form-group">
+    <div class="ordermain">
       <label class="col-md-4 control-label" for="toppings">Choose toppings</label>
-      <div class="col-md-4">
-        {{#each ingredient}}
-        <div class="checkbox">
-        <label for={{this.name}}>
-        <input name="toppings" id={{this.name}} value={{this.price}} type="checkbox">
-            {{this.name}}, ${{this.price}}
-          </label>
+      <div class="form-group sectionwrapper">
+
+        <div class="col-md-4">
+          {{#each ingredient}}
+          <div class="checkbox">
+            <label for={{this.name}}>
+              <input name="{{this.name}}" class=quant"{{this.quantity}}" value="{{this.price}}" type="checkbox">
```

Comment:
The stock is getting quoted which may be breaking the ability to not allow ordering out of stock ingredients.